Dockerfile.base

```diff
@@ -38,76 +38,46 @@ RUN apt-get update -qq && apt-get install -y -qq    \
                     clang-tidy                      \
                     jq                              \
                     kmod                            \
```

Comment:
I _think_ these two can also go, though not 100% sure we're not using `jq` in some scripts... (I see it in `roll_release.sh` but that isn't run in CI).